### PR TITLE
Fix checkbox rendering

### PR DESCRIPTION
**Pull request by dakota**
This fixes the checkbox rendering as reporting in #9.

### CODE REVIEW

File: src/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -67,7 +67,8 @@ public function create($model = null, array $options = [])
             $options['templates'] += [
                 'label' => '<label class="' . $options['horizontal']['left'] . '"{{attrs}}>{{text}}</label>',
                 'formGroup' => '{{label}}<div class="' . $options['horizontal']['right'] . '">{{input}}</div>',
-                'checkboxFormGroup' => '<div class="' . $options['horizontal']['combined'] . '">{{label}}</div>',
+                'checkboxFormGroup' => '<div class="' . $options['horizontal']['combined'] . '">
+                <div class="checkbox">{{label}}</div></div>',
```
Comment (jadb) [inline src/View/Helper/FormHelper.php]: Maybe better to use string concatenation when splitting the string?

File: src/View/Helper/FormHelper.php [hunk 2]
```diff
@@ -162,8 +163,12 @@ public function input($fieldName, array $options = [])
                 $this->templates(compact('input'));
         }
 
+        if ($options['type'] !== 'checkbox' && $options['type'] !== 'radio') {
```
Comment (jadb) [inline src/View/Helper/FormHelper.php]: How about:

```php
if (!in_array($options['type'], ['checkbox', 'radio]))
```

**Comment by jadb** [comment]
Looks good, thanks! :+1:

**Comment by dakota** [comment]
Updated based on your comments/suggestions @jadb

**Pull request merged**